include/mppic/critic_manager.hpp

```diff
@@ -1,38 +1,30 @@
 #ifndef MPPIC__CRITIC_MANAGER_HPP_
 #define MPPIC__CRITIC_MANAGER_HPP_
 
-#include <xtensor/xtensor.hpp>
 #include <pluginlib/class_loader.hpp>
+#include <xtensor/xtensor.hpp>
 
-#include "rclcpp_lifecycle/lifecycle_node.hpp"
 #include "geometry_msgs/msg/pose_stamped.hpp"
 #include "geometry_msgs/msg/twist.hpp"
 #include "geometry_msgs/msg/twist_stamped.hpp"
-#include "nav_msgs/msg/path.hpp"
 #include "nav2_costmap_2d/costmap_2d_ros.hpp"
+#include "nav_msgs/msg/path.hpp"
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
 
 #include "mppic/critic_function.hpp"
 #include "mppic/utils.hpp"
 
-namespace mppi
-{
+namespace mppi {
```

Comment:
We can leave this as is in this repo if you like, but just an FYI that cpplint / uncrustry styling say that this should be a new line (and at he bottom of files there should be 2 spaces between the comment for end namespace). I went through all of the files and manually changed this to comply, but if you prefer this we can keep it, but I will have to change this back for Nav2 eventually

Reply:
It's clang-format. I'll change all styling of the repo to cpplint/uncrustify

Reply:
Done